compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -130,6 +139,39 @@ buildDar service pkgConf@PackageConfigFields {..} ifDir dalfInput = do
                          dataFiles
                          ifaces
 
+-- | Write interface files and hie files to the location specified by the given options.
+writeIfacesAndHie ::
```

Comment:
This is unrelated to this PR and was literally copied from `ghcide` to match https://github.com/digital-asset/ghcide/pull/112 which I noticed while I was touching this code.